src/context/ChainApiContext.tsx

```diff
@@ -12,6 +13,8 @@ const initialState: ChainApiContext = {
   wsConnectionIndex: 0,
 };
 
+keyring.loadAll({store: keyringStore, type: 'sr25519'});
```

Comment:
We can not load the keyring every time on network change since this is not allowed. So it can be loaded globally and just change the `ss58Format` on network change.

But this also means that on RN hot reload there will be an error 🤔

Reply:
can we do this on an effect inside `ChainApiContextProvider`, maybe that would fix the problem with RN hot reload.

```
useEffect(() => {
  keyring.loadAll({store: keyringStore, type: 'sr25519'});
}, [])
```